staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -876,6 +877,19 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 		// override status subresource values
 		// shallow copy
 		statusScope := *requestScopes[v.Name]
+		statusScope.FieldManager, err = fieldmanager.NewDefaultCRDFieldManager(
```

Comment:
Contain in a `if utilfeature.DefaultFeatureGate.Enabled(features.ServerSideApply) {`?

Reply:
Thanks for spotting it!